src/pages/settings/Profile/ProfilePage.js

```diff
@@ -97,9 +131,23 @@ function ProfilePage(props) {
         },
     ];
 
-    useEffect(() => {
-        App.openProfile(props.currentUserPersonalDetails);
```

Comment:
Please make sure that this doesn't cause regression of user timezone info not sending to server.

Reply:
Other than this api, I don't see anywhere in the app where user sends local timezone info to server.
Please restore this back.

Reply:
Right, done